src/ovirtapi/transform.js

```diff
@@ -327,9 +327,7 @@ const VmStatistics = {
         stat.values.value &&
         (stat.name === 'disks.usage'
           ? stat.values.value[0] && stat.values.value[0].detail
-          : stat.values.value.length === 1
-            ? stat.values.value[0].datum
-            : stat.values.value.map(value => value.datum))
+          : stat.values.value.map(value => value.datum))
```

Comment:
It seems the API changed or is inconsistent. According to the comment above the API used to return a single object instead of a single element array.
I like that we treat all cases uniformly (as an array). Perhaps we could also check for the (legacy?) scenario with single element without array and wrap it into array?

Reply:
I made changes there, now it's array or undefined, please let me know WDYT